### PR TITLE
Fix: WarpX Version

### PR DESCRIPTION
**Pull request by ax3l**
Fix the WarpX version info.

- [x] test on app
- [x] test in Python

Introduced in #3933 for app and #3474 for Python bindings.

### CODE REVIEW

File: Source/Python/pyWarpX.cpp [hunk 1]
```diff
@@ -71,8 +71,8 @@ PYBIND11_MODULE(PYWARPX_MODULE_NAME, m) {
 
     // API runtime version
     //   note PEP-440 syntax: x.y.zaN but x.y.z.devN
-#ifdef PYIMPACTX_VERSION_INFO
-    m.attr("__version__") = MACRO_STRINGIFY(PYIMPACTX_VERSION_INFO);
+#ifdef pyWarpX_VERSION_INFO
```
Comment (dpgrote) [inline Source/Python/pyWarpX.cpp]: Should this be all caps?

Comment (ax3l) [reply Source/Python/pyWarpX.cpp:74]: Yes, but gotta change it in the CMake file as well.

Comment (ax3l) [reply Source/Python/pyWarpX.cpp]: Good catch!

Let me also update `setup.py`.

File: Source/Python/pyWarpX.cpp [hunk 2]
```diff
@@ -71,8 +71,8 @@ PYBIND11_MODULE(PYWARPX_MODULE_NAME, m) {
...
+    m.attr("__version__") = MACRO_STRINGIFY(pyWarpX_VERSION_INFO);
```
Comment (ax3l) [inline Source/Python/pyWarpX.cpp]:
```suggestion
#ifdef PYWARPX_VERSION_INFO
    m.attr("__version__") = MACRO_STRINGIFY(PYWARPX_VERSION_INFO);
```

**Review by dpgrote — APPROVED** [review]
This looks good!

**Pull request merged**